mlflow/server/handlers.py

```diff
@@ -1776,6 +1777,22 @@ def _create_registered_model():
         description=request_message.description,
     )
     response_message = CreateRegisteredModel.Response(registered_model=registered_model.to_proto())
+
+    try:
+        dispatch_webhook(
+            event=WebhookEvent.REGISTERED_MODEL_CREATED,
+            payload={
+                "name": request_message.name,
```

Comment:
nit: Shall we define a dataclass (or proto) for webhook event payload for discoverability of the payload schemas, and potentially users can define their endpoint using the payload schema class?

Reply:
Got it, I mentioned proto because users might want to use a different language for developing their webhook server